### DISCUSSION TITLE
Difference from software 1106 and 1326 audi a3 mib2

### DISCUSSION THREAD

**Discussion by djmarkino94** [Mr-MIBonk/M.I.B._More-Incredible-Bash#89] [General]
Hi I have a question, I happily use your patch on audi a3 mib2 with software version 1106. I wanted to ask something. is it possible to install 1326 after having already patched the mib? what improvements would the 1326 bring? Thanks so much for your work.

**Comment by harman-f** [comment]
Patch will be gone after FW update.
There are no changelogs available.
Normally it is better to stay on current FW if no issues are present.
Just patch waht you have.

**Reply by djmarkino94** [reply]
ok thanks cleared up any doubts, i was just trying to figure out if there was a way to get back to seeing google earth without getting people to rip you off your money, i thought it needed an update
